SilKit/cmake/SilKitTest/SilKitTestMain.cpp

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2024 Vector Informatik GmbH
+//
+// SPDX-License-Identifier: MIT
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+#include "silkit/SilKitVersion.hpp"
+
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    std::cout << "Running main() from SIlKitTestMain.cpp" << std::endl;
+
+    testing::InitGoogleMock(&argc, argv);
+
+#ifndef SIL_KIT_TEST_MAIN_DO_NOT_PRINT_VERSION
+    std::cout << "Running with SIL Kit version '" << SilKit::Version::String() << "' and git hash '"
```

Comment:
could we use a gtest stream instead of std::cout instead? or is this negligable overhead

Reply:
This is neglegible. In fact the `main` function provided by [`gmock_main`](https://github.com/google/googletest/blob/main/googlemock/src/gmock_main.cc) uses `std::cout` as well ([`gtest_main`](https://github.com/google/googletest/blob/main/googletest/src/gtest_main.cc) uses `std::printf` ❓)